### ISSUE TITLE
Build failed with: Could not find com.vanniktech:gradle-maven-publish-plugin:0.9.0-SNAPSHOT.

### ISSUE THREAD

**Issue by eboudrant** [airbnb/epoxy#959]
With the latest master : 
```
$ ./gradlew check
Configuration on demand is an incubating feature.

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'epoxy'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not find com.vanniktech:gradle-maven-publish-plugin:0.9.0-SNAPSHOT.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/vanniktech/gradle-maven-publish-plugin/0.9.0-SNAPSHOT/maven-metadata.xml
       - https://dl.google.com/dl/android/maven2/com/vanniktech/gradle-maven-publish-plugin/0.9.0-SNAPSHOT/gradle-maven-publish-plugin-0.9.0-SNAPSHOT.pom
       - https://dl.google.com/dl/android/maven2/com/vanniktech/gradle-maven-publish-plugin/0.9.0-SNAPSHOT/gradle-maven-publish-plugin-0.9.0-SNAPSHOT.jar
       - https://jcenter.bintray.com/com/vanniktech/gradle-maven-publish-plugin/0.9.0-SNAPSHOT/maven-metadata.xml
       - https://jcenter.bintray.com/com/vanniktech/gradle-maven-publish-plugin/0.9.0-SNAPSHOT/gradle-maven-publish-plugin-0.9.0-SNAPSHOT.pom
       - https://jcenter.bintray.com/com/vanniktech/gradle-maven-publish-plugin/0.9.0-SNAPSHOT/gradle-maven-publish-plugin-0.9.0-SNAPSHOT.jar
       - https://oss.sonatype.org/service/local/repositories/snapshots/content/com/vanniktech/gradle-maven-publish-plugin/0.9.0-SNAPSHOT/maven-metadata.xml
       - https://oss.sonatype.org/service/local/repositories/snapshots/content/com/vanniktech/gradle-maven-publish-plugin/0.9.0-SNAPSHOT/gradle-maven-publish-plugin-0.9.0-SNAPSHOT.pom
       - https://oss.sonatype.org/service/local/repositories/snapshots/content/com/vanniktech/gradle-maven-publish-plugin/0.9.0-SNAPSHOT/gradle-maven-publish-plugin-0.9.0-SNAPSHOT.jar
       - https://plugins.gradle.org/m2/com/vanniktech/gradle-maven-publish-plugin/0.9.0-SNAPSHOT/maven-metadata.xml
       - https://plugins.gradle.org/m2/com/vanniktech/gradle-maven-publish-plugin/0.9.0-SNAPSHOT/gradle-maven-publish-plugin-0.9.0-SNAPSHOT.pom
     Required by:
         project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3s
```

Am I doing something wrong?

**Comment by eboudrant** [comment]
According https://github.com/vanniktech/gradle-maven-publish-plugin/issues/54 we could update to `gradle-maven-publish-plugin:0.10.0`

**Comment by elihart** [comment]
yeah, I  think we should upgrade to `0.10.0`

**Comment by githubashutoshsoni** [comment]
Yeah I am getting the same issue. How do I resolve it? Which files to update?

**Comment by Aswar-Group** [comment]
@githubashutoshsoni  go-to  Epoxy  project `gradle.build`  file and then update 

`    classpath 'com.vanniktech:gradle-maven-publish-plugin:0.9.0-SNAPSHOT'`

to 
`  classpath 'com.vanniktech:gradle-maven-publish-plugin:0.9.0 '`

or 
` classpath 'com.vanniktech:gradle-maven-publish-plugin:0.10.0 '`

and then sync your project again

**Comment by Ezaldeen99** [comment]
@githubashutoshsoni go-to Epoxy project `gradle.build `file and then update
` classpath 'com.vanniktech:gradle-maven-publish-plugin:0.9.0-SNAPSHOT' `

to 

`classpath 'com.vanniktech:gradle-maven-publish-plugin:0.9.0 ' `
or 
`classpath 'com.vanniktech:gradle-maven-publish-plugin:0.10.0 '`

both will do fine 

 and then sync your project again

**Comment by eboudrant** [comment]
If I just update the version `gradle-maven-publish-plugin` I have this error: 
```
* What went wrong:
A problem occurred configuring project ':epoxy-adapter'.
> SoftwareComponentInternal with name 'release' not found.
```
I must me missing something, may be from the `local.properties`?

**Comment by githubashutoshsoni** [comment]
I updated to  `classpath 'com.vanniktech:gradle-maven-publish-plugin:0.10.0'`
 as suggeseted by @Ezaldeen99  now I have this error.
`epoxy master failed`
`Caused by: org.gradle.api.UnknownDomainObjectException: SoftwareComponentInternal with name 'release' not found.`

**Comment by DastanIqbal** [comment]
this worked `classpath 'com.vanniktech:gradle-maven-publish-plugin:0.9.0'`

**Comment by Ezaldeen99** [comment]
@githubashutoshsoni  can you please use the 
`classpath 'com.vanniktech:gradle-maven-publish-plugin:0.9.0 '`
and check

**Comment by elihart** [comment]
I've merged a changed to master to use `0.9.0` for now. I'll look into why 0.10.0 is not working
